daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -191,9 +191,8 @@ func subvolCreate(path, name string) error {
 		args.name[i] = C.char(c)
 	}
 
-	_, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
-		uintptr(unsafe.Pointer(&args)))
-	if errno != 0 {
+	if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
```

Comment:
Thanks, everyone.
:cry: ,it is diffcult to fit the golang usage and code readability.
It is better to close this PR, and keep the code as it is, I think.
If have any good method, I will change this code and the code in this picture: 
![image](https://cloud.githubusercontent.com/assets/20062886/26396190/0c9b4bd4-40a5-11e7-9d79-beaac2b08421.png)